modules/ansible/templates-ansible/configure-monitoring-sap/playbook-configure-monitoring-sap.yml.tftpl

```diff
@@ -0,0 +1,40 @@
+# ------------------------------------------------------------------------
+# This playbook uses the ibm.power_linux_sap collection. This collection is
+# available on ansible galaxy
+# https://galaxy.ansible.com/ui/repo/published/ibm/power_linux_sap/ and can
+# be installed using 'ansible-galaxy collection install ibm.power_linux_sap'
+# ------------------------------------------------------------------------
+
+---
+- name: SAP monitoring configuration
+  hosts: all
+  become: true
+  vars:
+    monitoring_host_ip: '${monitoring_host_ip}
```

Comment:
why is monitoring_host_ip required here ?
I don't think your playbook uses this